flexmeasures/data/models/planning/__init__.py

```diff
@@ -173,6 +173,18 @@ def persist_flex_model(self):
         """
         pass
 
+    def fetch_db_flex_config(self):
+        """Merge the flex-config from the db (from the asset and its ancestors) with the initialization flex-config.
+
+        Note that self.flex_context overrides db_flex_context (from the asset and its ancestors).
+        """
```

Comment:
Let's add a TODO here, as the flex model will also come from the db (in fact the PR doing that data migration should add also that missing part here)